libs/common/src/lib/navbar/navbar.component.html

```diff
@@ -264,7 +264,7 @@
               <span class="nav-link-title">{{ 'navbar.nua-keys' | translate }}</span>
             </a>
           </div>
-          @if (isBillingEnabled | async) {
+          @if ((isBillingEnabled | async) && !noStripe) {
```

Comment:
Should the "Payment & Subscription" link in the topbar be also hidden ?

Reply:
fixed